README.md

```diff
@@ -261,73 +259,350 @@ The SDK provides a simple interface for all the [media APIs mentioned here](http
+Accepts an object of class `TagsRequest` specifying the parameters to be used to add tags. All parameters specified in the [documentation here](https://docs.imagekit.io/api-reference/media-api/add-tags-bulk) can be passed via their setter functions to get the results.
+
+```java
+List<String> fileIds = new ArrayList<>();
+fileIds.add("FileId");
+List<String> tags = new ArrayList<>();
+tags.add("tag-to-add-1");
+tags.add("tag-to-add-2");
+ResponseMetaData resultTags=ImageKit.addTags(new TagsRequest(fileIds, tags));
 
-```cs
-PurgeAPIResponse resp = await imagekit.PurgeCacheAsync("full_url");
 ```
 
-**7. Purge Cache Status**
+**7. Remove tags**
 
-Get the purge cache request status using the request ID returned when a purge cache request gets submitted as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/purge-cache-status)
+Accepts an object of class `TagsRequest` specifying the parameters to be used to remove tags. All parameters specified in the [documentation here](https://docs.imagekit.io/api-reference/media-api/remove-tags-bulk) can be passed via their setter functions to get the results.
 
-```cs
-PurgeCacheStatusResponse resp = await imagekit.GetPurgeCacheStatusAsync("cache_request_id");
+```java
+List<String> fileIds = new ArrayList<>();
+fileIds.add("FileId");
+List<String> tags = new ArrayList<>();
+tags.add("tag-to-remove-1");
+tags.add("tag-to-remove-2");
+ResponseMetaData resultTags=ImageKit.removeTags(new TagsRequest(fileIds, tags));
+
+```
+
+**8. Remove AI tags**
+
+Accepts an object of class `AITagsRequest` specifying the parameters to be used to remove AI tags. All parameters specified in the [documentation here](https://docs.imagekit.io/api-reference/media-api/remove-aitags-bulk) can be passed via their setter functions to get the results.
+
+```java
+List<String> fileIds = new ArrayList<>();
+fileIds.add("629f3de17eb0fe4053615450");
+List<String> aiTags = new ArrayList<>();
+aiTags.add("Rectangle");
+AITagsRequest aiTagsRequest =new AITagsRequest();
+aiTagsRequest.FileIds=fileIds;
+aiTagsRequest.AITags=aiTags;
+ResponseMetaData resultTags = ImageKit.getInstance().removeAITags(aiTagsRequest);
+
+```
+
+**9. Delete File**
+
+Accepts the file ID and delete a file as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/delete-file).
+
+```java
+String fileId="your-file-id";
+ResponseMetaData result=ImageKit.deleteFile(fileId);
+
+```
+
+**10. Delete FileVersion**
+
+Accepts an object of class `DeleteFileVersionRequest` specifying the parameters to be used to delete file version. All parameters specified in the [documentation here](https://docs.imagekit.io/api-reference/media-api/delete-file-version) can be passed via their setter functions to get the results.
+
+```java
+DeleteFileVersionRequest deleteFileVersionRequest = new DeleteFileVersionRequest();
+deleteFileVersionRequest.FileId="629d95278482ba129fd17c97";
```

Comment:
What is the convention of the setter and getter functions in `.NET`? In Java for instance, we have `setFileId` and `setVersionId`. Do we not use the `set` prefix in `.NET`?

Reply:
in .net we don't need setter and getter functions. .net object properties has feature to auto support getter & setter.